package.json

```diff
@@ -9,7 +9,7 @@
     "bootstrap": "^5.3.5",
     "@hotwired/turbo-rails": "^8.0.13",
     "clipboard": "^2.0.11",
-    "datatables.net": "^2.2.1",
+    "datatables.net": "^2.3.0",
```

Comment:
This is a result of merging `-bs5` PR which directly depends on `datatables.net` `2.3.0`. `^2.2.1` is _satisfied_ by `2.3.0`, but the PR that bumps this requirement got closed by Dependabot.